etc/bento_post_config.bash

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env bash
 
 if [[ "$BENTO_GATEWAY_USE_TLS" == 'true' || "$BENTO_GATEWAY_USE_TLS" == '1' ]]; then
-  KC_HOSTNAME_STRICT_HTTPS='true'
+  KC_HOSTNAME=${BENTOV2_AUTH_DOMAIN}
   KC_HTTP_ENABLED='false'
   KC_HTTPS_CERTIFICATE_FILE=/run/secrets/keycloak-cert-file
   KC_HTTPS_CERTIFICATE_KEY_FILE=/run/secrets/keycloak-cert-key-file
   KC_PROXY='passthrough'
 else
   # Disable TLS in keycloak
-  KC_HOSTNAME_STRICT_HTTPS='false'
+  KC_HOSTNAME=https://${BENTOV2_AUTH_DOMAIN}  # full URL with HTTPS when KC_HTTP_ENABLED=true
   KC_HTTP_ENABLED='true'
   KC_PROXY='edge'
+  KC_PROXY_HEADERS=xforwarded
```

Comment:
maybe a comment explaining this? why `xforwarded` and not the standardized `forwarded`?

Reply:
Done. NGINX has yet to support `forwarded` (RFC7239).